soh/src/code/z_parameter.c

```diff
@@ -3084,7 +3084,10 @@ s32 Health_ChangeBy(PlayState* play, s16 healthChange) {
 }
 
 void Rupees_ChangeBy(s16 rupeeChange) {
-    gSaveContext.rupeeAccumulator += rupeeChange;
+    if (gPlayState == NULL)
+        gSaveContext.rupees += rupeeChange;
+    else
+        gSaveContext.rupeeAccumulator += rupeeChange;
```

Comment:
```suggestion
    if (gPlayState == NULL) {
        gSaveContext.rupees += rupeeChange;
    } else {
        gSaveContext.rupeeAccumulator += rupeeChange;
    }
```
unless you're doing the full statement on the same line it's always a good idea to use braces. https://www.imperialviolet.org/2014/02/22/applebug.html